### PR TITLE
docs: Extend LRP guide with troubleshooting section

### PR DESCRIPTION
**Pull request by aditighag**

### CODE REVIEW

**Review by bimmlerd — APPROVED** [review]
CODEOWNERS lgtm

File: Documentation/network/kubernetes/local-redirect-policy.rst
Comment (ghost) [inline Documentation/network/kubernetes/local-redirect-policy.rst:411]:
```suggestion
Troubleshooting
~~~~~~~~~~~~~~~
```
I'd expect the troubleshooting section to be a nested section, not just a bullet point. But given the deployment instructions are also bullet points, I won't block on it.

File: Documentation/network/kubernetes/local-redirect-policy.rst
Comment (ghost) [inline Documentation/network/kubernetes/local-redirect-policy.rst]:
```suggestion
          As a workaround, you can restart the node-local DNS DaemonSet pods. File a `GitHub issue <https://github.com/cilium/cilium/issues/new/choose>`_
```
small suggestion

**Review by ghost — APPROVED** [review]
I left some suggestions, but won't block on them.

**Review by ysksuzuki — APPROVED** [review]

File: Documentation/network/kubernetes/local-redirect-policy.rst
Comment (aditighag) [reply Documentation/network/kubernetes/local-redirect-policy.rst:411]: Tried this, the troubleshooting section sort of merges in the block of text above. 

>But given the deployment instructions are also bullet points, I won't block on it.

Right! It's aligned to the other bullet points.

File: Documentation/network/kubernetes/local-redirect-policy.rst
Comment (aditighag) [reply Documentation/network/kubernetes/local-redirect-policy.rst]: Cool... applied!

**Comment by aditighag** [comment]
Documentation only changes, marking it as ready for merge.

**Pull request merged**
